src/main/java/mate/controller/CreateCarController.java

```diff
@@ -0,0 +1,42 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.Collections;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/cars/add")
+public class CreateCarController extends HttpServlet {
+    private static final Injector INJECTOR = Injector.getInstance("mate");
+    private final CarService carService = (CarService) INJECTOR.getInstance(CarService.class);
+    private final ManufacturerService manufacturerService
+            = (ManufacturerService) INJECTOR.getInstance(ManufacturerService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/car-form.jsp").forward(req, resp);
```

Comment:
better divide your jsp to different packages 
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/cars/add.jsp").forward(req, resp);
```
check it everywhere